### PR TITLE
fix(search_best_perf): add client encryption

### PR DESCRIPTION
**Pull request by aleksbykov**
Add client encryption to search-best-performance test config file

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: test-cases/performance/perf-search-best-throughput-config.yaml [hunk 1]
```diff
@@ -32,3 +32,5 @@ email_recipients: ['scylla-perf-results@scylladb.com']
 store_perf_results: false
 # tests are running on aws. The loader uses the shard-aware driver
 ami_id_loader: 'scylla-qa-loader-ami-v19'
+
+client_encrypt: true
```
Comment (roydahan) [inline test-cases/performance/perf-search-best-throughput-config.yaml:36]: @aleksbykov I hoped to controlled it from the pipeline.
By default it should be probably set to false, but when checking it, set to true.

Comment (aleksbykov) [reply test-cases/performance/perf-search-best-throughput-config.yaml:36]: Ok. i will redo

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by roydahan — APPROVED** [review]

**Comment by roydahan** [comment]
backported to v14 and also changed the job to run from official repo and v14 branch.

**Pull request merged**